isis/src/control/objs/BundleSolutionInfo/BundleSolutionInfo.h

```diff
@@ -250,7 +257,8 @@ namespace Isis {
       QString             m_name;                        //!< Name of the bundle. Defaults to the id
       QString             m_runTime;                     //!< Run time of the bundle adjustment
       FileName           *m_inputControlNetFileName;     //!< Input control network file name
-      Control            *m_outputControl;               //!< Output control
+      Control *           m_outputControl;               //!< Output control
```

Comment:
Why is the pointer in the wrong location

Reply:
@SgStapleton Fixed it.